### PR TITLE
py-bbp-workflow: bump

### PR DESCRIPTION
**Pull request by genric**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-xarray/package.py
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-xarray/package.py:1]: I assume this is straight from upstream?

**Review by matz-e — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-xarray/package.py
Comment (genric) [reply var/spack/repos/builtin/packages/py-xarray/package.py:1]: It is still in review...

Comment (matz-e) [reply var/spack/repos/builtin/packages/py-xarray/package.py:1]: OK, I think we should be good to merge then. Thank you for updating upstream!

Comment (genric) [reply var/spack/repos/builtin/packages/py-xarray/package.py:1]: So is this resolved then?

Comment (matz-e) [reply var/spack/repos/builtin/packages/py-xarray/package.py:1]: Yes

**Pull request merged**